src/app/parcels/parcelsTable/filters.ts

```diff
@@ -58,8 +54,8 @@ const familySearch: ParcelsFilterMethod<string> = (query, state) => {
     return query.eq("family_count", Number(state));
 };
 
-const phoneSearch: ParcelsFilterMethod<string> = (query, state) =>
-    query.ilike("client_phone_number", `%${state}%`);
+const phoneSearch: ParcelsFilterMethod<string> =
```

Comment:
I feel like if we change the `phoneNumberMethodHelper` to be `phoneNumberSearch`, we could rename this to `parcelsPhoneNumberSearch` which I think is a bit clearer. Similarly for the others